File: redsheet-portal/src/app/database-models/v-nano-our-batna.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vNanoOurBatna {
    constructor(vNanoOurBatna?: vNanoOurBatna )  {
        let properties = [ "nanoOurBatnaId", "nanoOurBatnaValue", "projectModuleId",];
        for (let property of properties)
            if (vNanoOurBatna && vNanoOurBatna[property])
                this[property] = vNanoOurBatna[property];
    }
 
	nanoOurBatnaId : number =   0 ;
 
    @required()
    @maxLength(1000)
	nanoOurBatnaValue : string =   undefined;
 
    @range(1,2147483647)
	projectModuleId : number =   undefined;


}

File: RedSheet.DbEntities/Models/vNanoOurBatna.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vNanoOurBatnas", Schema = "dbo")]
	[Serializable]
	public partial class vNanoOurBatna  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int NanoOurBatnaId { get; set; } 
 
	[Required] 
	[MaxLength(1000)]
	public string NanoOurBatnaValue { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ProjectModuleId { get; set; } 

	public vNanoOurBatna()
	{
    }

    }
}

